src/xo-app/home/host-item.js

```diff
@@ -115,11 +116,11 @@ export default class HostItem extends Component {
               &nbsp;&nbsp;
               <Tooltip
                 content={isEmpty(host.current_operations)
-                  ? _(`powerState${host.power_state}`)
-                  : <div>{_(`powerState${host.power_state}`)}{' ('}{map(host.current_operations)[0]}{')'}</div>
+                  ? toolTipContent
+                  : <div>{toolTipContent}{' ('}{map(host.current_operations)[0]}{')'}</div>
                 }
               >
-                {isEmpty(host.current_operations)
+                {isEmpty(host.current_operations) && (host.power_state !== 'Running' || host.enabled)
```

Comment:
We want an orange icon when !isEmpty(host.current_operations) or (host.power_state !== 'Running' && !host.enabled)
So we don't want an orange icon when !(isEmpty(host.current_operations) || (host.power_state !== 'Running' && !host.enabled)) <=> isEmpty(host.current_operations) && (host.power_state !== 'Running' || host.enabled))
I did it to change as little as possible the code which was already here